lib/resources/r4/related_person_patch.yaml

```diff
@@ -189,7 +228,7 @@ operations:
     note: >
       <ul>
         <li>Must contain the `system`, `use`, and `value` fields.</li>
-        <li>When specifying a `period`, the `start` and `end` fields must include a time component with a timezone.</li>
+       <li>When specifying a period, the fields must include a time component with a timezone.</li>
```

Comment:
nit: add the removed space

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/650/commits/43924ca7f7de4aaa0e48bf96583ee7ec9d1a2125#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR231